src/components/task/existing-task/edit-task/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from "react";
+import propTypes from "prop-types";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useDispatch } from "react-redux";
+import TextArea from "components/base/text-area";
+import Button from "components/base/button";
+import Image from "components/base/image";
+import {
+  TASK_TEXTAREA_NUM_OF_ROW,
+  TYPE_BUTTON,
+  TITLE_FIELD_NAME_ATTRIBUTE,
+  PATH_CHECK_ICON,
+  ALT_CHECK_ICON_TAG,
+  PATH_DELETE_ICON,
```

Comment:
use common prefix to group together similar constants

```suggestion
  ICON_CHECK,
  ALT_CHECK_ICON_TAG,
  ICON_DELETE,
```